crypto/mime.go

```diff
@@ -46,6 +46,44 @@ func (keyRing *KeyRing) DecryptMIMEMessage(
 	callbacks.OnEncryptedHeaders("")
 }
 
+type MIMEMessage struct {
+	Headers      []string
+	BodyMIMEType string
+	BodyContent  string
+	Attachments  []*Attachment
+}
+
+type Attachment struct {
+	Header  string
+	Content []byte
+}
+
+// DecryptMIMEMessageSynchronously decrypts a MIME message.
+func (keyRing *KeyRing) DecryptMIMEMessageSynchronously(
```

Comment:
`DecryptMIMEMessageSync`, maybe? (At least that's the convention in Node.js, idk if there's another convention in Kotlin or Android?)

Reply:
I don't think there is such a convention in android, but Sync is shorter while still being clear so it makes sense